src/Build/BackEnd/Components/RequestBuilder/TaskHost.cs

```diff
@@ -34,7 +36,7 @@ internal class TaskHost :
 #if FEATURE_APPDOMAIN
         MarshalByRefObject,
 #endif
-        IBuildEngine7
+        IBuildEngine7, IRarBuildEngine
```

Comment:
There is also problem with the fact that the Tasks project is build against .NET Standard and not .NET Core 2.1, which caused some issues, because not all necessary interfaces are there.

Reply:
.NET Standard 2.0 I gather? I think .NET Standard 2.1 is essentially .NET Core 2.1.

Reply:
Yes, that is the issue. Related to this conservation: https://github.com/dotnet/msbuild/pull/5613/files#r477380003. Will continue with that conservation and close this one.